frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -356,6 +382,13 @@ const SaveAndDiscardButtons = ({
         variant="filled"
         data-testid="strategy-form-submit-button"
         className="strategy-form-submit-button"
+        styles={
+          isInSidebar
+            ? {
+                root: { flex: 1 },
+              }
+            : undefined
+        }
```

Comment:
I think the `fullWidth` prop allow you to avoid the dreaded `styles={{}}` here

https://v6.mantine.dev/core/button/#full-width-and-overflow

Reply:
Alas, `fullWidth` sets `width: 100%` but I really want the button to take up the rest of the row